Inputting/InputDefinitions/AnalogInputDefinition.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Inputting.InputDefinitions
+{
+    /// <summary>
+    /// An analog input is an input that has a float value from 0 to 1 on how far it is being pressed.
```

Comment:
how "far"? sounds odd. I think I wrote "how strong" somewhere else. Someone help me to grammer pls

Reply:
"...that has a float value from 0 to 1 indicating the distance away from the neutral input as a fraction of the maximum allowable by the controller"

Reply:
those were words